.gcloudignore

```diff
@@ -20,3 +20,4 @@ yarn-error.log
 .git
 .hg
 .svn
+.env
```

Comment:
@ChrisAntaki Sorry, forgot to add this earlier. I tested re-deploying to GCP and realized that the `.env` file went up too! Oops.